server/main.py

```diff
@@ -23,6 +23,7 @@
 @asynccontextmanager
 async def lifespan(app:FastAPI):
     await database.connect()
+    await database.execute("create table if not exists user( id integer primary key autoincrement, username text not null, password text not null, key blob default None,cookie text unique default None) ;")
```

Comment:
You can make this (and other SQL) more readable by making them multiline. 

It is generally easier to find bugs (typo, missing comma etc.) if the code is more readable.

```suggestion
	await database.execute("""
		create table if not exists user(
			id integer primary key autoincrement,
			username text not null,
			password text not null,
			key blob default None,
			cookie text unique default None
		);
	""")


```